### ISSUE TITLE
Run linter as Python package FileNotFoundError

### ISSUE THREAD

**Issue by larsdekwant** [cpp-linter/cpp-linter-action#67]
We were using the Windows linter in our project with fix from issue #50 . However, with the latest merge, running the linter as a Python package throws the following error: `FileNotFoundError: [WinError 2] The system cannot find the file specified`

The first action that failed for us, can be found here: https://github.com/red-panda-productions/DAISI/runs/6963055842?check_suite_focus=true

We have fixed the issue by cloning an older version of the master branch, but just wanted to let you know about this issue!

**Comment by 2bndy5** [comment]
After researching the point of error, I found that red-panda-productions/DAISI workflow is improperly passing the version number to the `version` arg. When running on a Windows runner, the only ways to invoke a clang tool executable is to use
- `clang-tidy` / `clang-format` (this invokes whatever the default provided version is for the runner)
- `path\to\bin\clang-tidy.exe` / `path\to\bin\clang-format.exe` (this is used to specify a specifically installed version)

As stated in the README, Windows runners now come with v13 of the clang tools. If you want to use a specifically installed version on Windows then you have to pass the path it was in stalled in. (see example in README). Our python code will then check if the path has a `bin` folder and execute `path\to\bin\clang-<tool>.exe` (only the `path\to` part needs to be passed to the `version` arg.

To summarize, the working state of the red-panda-productions/DAISI workflow passes `12` to the version arg, but our action will only execute `clang-<tool>`. This ends up using v13 because (from what I can tell) v12 is not installed in the runner's environment, and calling `clang-<tool>` (without the `.exe`) invokes whatever executable is found in the OS's `PATH` env variable.

**Comment by 2bndy5** [comment]
My faulty logic revision in #66 (now fixed by #68) allowed this improper usage of the `version` arg to surface as an error.
